packages/js/product-editor/src/hooks/use-product-variations-helper.ts

```diff
@@ -21,20 +21,18 @@ import { DEFAULT_VARIATION_PER_PAGE_OPTION } from '../constants';
 async function getDefaultVariationValues(
 	productId: number
 ): Promise< Partial< Omit< ProductVariation, 'id' > > > {
-	const attributes = (
-		( await resolveSelect( 'core' ).getEntityRecord(
+	try {
+		const { attributes } = ( await resolveSelect( 'core' ).getEntityRecord(
```

Comment:
I still think this request is not necessary here since it's been made in line 80 and there you can get both `const { status: lastStatus, attributes } = await ...` also you can do `getEditedEntityRecord< Product >( ... )` instead of `getEditedEntityRecord( ... ) as Product` too